documentation/dotnet-trace-instructions.md

```diff
@@ -66,7 +66,7 @@ GCSampledObjectAllocationHigh   |            200000 | Enables allocation samplin
 GCHeapSurvivalAndMovement       |            400000 | Enables events associate with object movement or survival with each GC.
 GCHeapCollect                   |            800000 | Triggers a GC. Can pass a 64 bit value that will be logged with the GC Start event so you know which GC you actually triggered.
 GCHeapAndTypeNames              |           1000000 | Indicates that you want type names looked up and put into the events (not just meta-data tokens).
-GCHeapSnapshot                  |           1980001 | This provides the flags commonly needed to take a heap .NET Heap snapshot with ETW.
+GCHeapSnapshot                  |           1980001 | This provides the flags commonly needed to take a heap .NET Heap snapshot with ETW.<br>This is equivalent to `GC+Type+GCHeapDump+GCHeapCollect+GCHeapAndTypeNames`
```

Comment:
This isn’t part of your change in this PR but should we say ETW here? Wouldn’t it be better if you say EventPipe instead?